content/documentation/validations/index.adoc

```diff
@@ -759,6 +760,39 @@ https://istio.io/docs/reference/config/security/authorization-policy/#Source[Def
 
 '''
 
+:numbered:
+=== This host has no matching entry in the service registry
```

Comment:
Does it need a code ?

Reply:
The codes PR wasn't merged yet. Rebased and changed.